src/main/java/com/ecore/roles/web/rest/UsersRestController.java

```diff
@@ -24,7 +21,7 @@ public class UsersRestController implements UsersApi {
     private final UsersService usersService;
 
     @Override
-    @PostMapping(
+    @GetMapping(
```

Comment:
In order to follow the Rest specification, of course, if the there isn’t any other security reason or URL length limitation